build.sbt

```diff
@@ -48,17 +47,10 @@ lazy val templateImports: Seq[String] = {
     "play.api.templates.PlayMagic._"
   )
 
-  val specificImports = PlayCrossCompilation.playVersion match {
-    case Play25 =>
-      Seq(
-        "_root_.play.twirl.api.TemplateMagic._"
-      )
-    case _ =>
-      Seq(
-        "_root_.play.twirl.api.TwirlFeatureImports._",
-        "_root_.play.twirl.api.TwirlHelperImports._"
-      )
-  }
+  val specificImports = Seq(
+    "_root_.play.twirl.api.TwirlFeatureImports._",
+    "_root_.play.twirl.api.TwirlHelperImports._"
+  )
 
   allImports ++ specificImports
```

Comment:
You can probably just collapse this into one Seq now, rather than constructing two separately and combining.

Reply:
Yeah I noticed that just after I submitted the PR and updated in latest commit.